docs/guides/latest/batch-backup/batch-backup.md

```diff
@@ -489,170 +506,450 @@ spec:
-        Path:                /var/www/html
-        Processing Time:     0:24
-        Total Size:          42.702 MiB
-        Uploaded:            42.645 MiB
-    Total Hosts:             1
-Events:
-  Type    Reason                   Age    From                      Message
-  ----    ------                   ----   ----                      -------
-  Normal  BackupSession Running    2m18s  BackupSession Controller  Backup job has been created succesfully/sidecar is watching the BackupSession.
-  Normal  BackupSession Running    2m18s  BackupSession Controller  Backup job has been created succesfully/sidecar is watching the BackupSession.
-  Normal  Host Backup Succeeded    88s    Status Updater            backup succeeded for host host-0 of "Deployment"/"wordpress".
-  Normal  Host Backup Succeeded    12s    Status Updater            backup succeeded for host host-0 of "AppBinding"/"sample-mysql".
-  Normal  BackupSession Succeeded  12s    BackupSession Controller  Backup session completed successfully
-```
-
-Now, we are going to verify whether the backed up data has been stored in the backend.
+apiVersion: stash.appscode.com/v1beta1
+kind: RestoreBatch
+metadata:
+  name: wordpress-restore
+  namespace: demo
+spec:
+  driver: Restic
+  repository:
+    name: gcs-repo
+  executionOrder: Sequential
+  members:
+  - target:
+      alias: db
+      ref:
+        apiVersion: appcatalog.appscode.com/v1alpha1
+        kind: AppBinding
+        name: wordpress-db
+      rules:
+      - snapshots: [latest]
+    task:
+      name: mysql-restore-v2020.07.09-beta.0
+  - target:
+      alias: app
+      ref:
+        apiVersion: apps/v1
+        kind: Deployment
+        name: wordpress-app
+      rules:
+      - paths:
+        - /var/www/html
+      volumeMounts:
+      - name: storage
+        mountPath: /var/www/html
+```
 
-**Verify Backup:**
+Here,
 
-When a backup session is completed, Stash will update the respective `Repository` to reflect the latest state of backed up data.
+- `spec.repository` specifies that we are restoring from the `gcs-repo` Repository.
+- `spec.executionOrder` specify that we want Stash to restore the component sequentially. Here, we want to restore the database first then we want to restore the wordpress deployment's data.
+- `spec.members` specify the targets to be restored. Each member may have the following sub-fields.
+  - `target.alias` specify the host identifier of the backed up data for this member. It must be the same as the `alias` used during backup.
+  - `target.ref` refers to the target to restore.
+  - `target.rules` specify the rules for restoring the data of this member. Here, we want to restore the `latest` snapshot for the database and the latest state of `/var/www/html` path for our WordPress deployment.
+  - `target.volumeMounts` specifies the volume mounts where the data will be restored.
+  - `task.name` refers to the `Task` object to use to restore the member in the Function-Task model.
```

Comment:
Should we also mention task.params ?

Reply:
No. This example does not utilize that field. `task.params` has been explained in concept docs.